nengo_gui/components/pointer.py

```diff
@@ -16,17 +15,31 @@ def __init__(self, obj, **kwargs):
         # the semantic pointer value as set by the user in the GUI
         # a value of 'None' means do not override
         self.override_target = None
-        self.vocab_in = obj.inputs[self.target][1]
+        # The white list indicates the networks whose user-defined
+        # over-ride value can be inserted on the input
+        # thus the value loops in from the output to the input.
+        # All other networks have their value inserted on the output
```

Comment:
Add in a mention that the whole point of looping in on the input is so that the actual neural activity of this module will be changed by setting its value, rather than just the output of the module.